src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/CatalogueSolutionsController.cs

```diff
@@ -131,9 +131,14 @@ public async Task<ActionResult<GetCatalogueSolutionOrderItemModel>> GetOrderItem
                 return Forbid();
             }
 
+            if (model is null)
```

Comment:
yeah I did consider that, I put it there to avoid throwing exceptions before authorisation etc had kicked it, but I think it's fair to move it to the top. Will do!